controller/monitor/manager.py

```diff
@@ -0,0 +1,42 @@
+from typing import Any, List
+from submodules.model.business_objects import monitor
+
+
+def monitor_all_tasks(project_id: str = None, only_running: bool = False) -> List[Any]:
+    return monitor.get_all_tasks(project_id, only_running)
+
+
+def cancel_all_running_tasks(project_id: str = None) -> None:
+    monitor.cancel_all_running_tasks(project_id)
+
+
+def cancel_upload_task(project_id: str = None, upload_task_id: str = None) -> None:
+    monitor.set_upload_task_to_failed(project_id, upload_task_id, True)
```

Comment:
usually we set with_commit=True (named variable) to prevent misunderstandings & later extension issues

Reply:
same for the other methods in the file

Reply:
For the manager methods not I would have thought, in the business object methods this behavior is there

Reply:
ah nvm misunderstood the question